clippy_utils/src/ty.rs

```diff
@@ -159,6 +159,16 @@ pub fn get_type_diagnostic_name(cx: &LateContext<'_>, ty: Ty<'_>) -> Option<Symb
     }
 }
 
+/// Returns true if `ty` is a type on which calling `Clone` through a function instead of
+/// as a method, such as `Arc::clone()`. Lints should avoid suggest replacing instances
```

Comment:
Is this sentence missing something at the end? Like "Returns true if `ty` is a type on which [...] **is considered idiomatic**" or something?

Reply:
Thanks for catching this.